api/src/pcapi/routes/pro/offerers.py

```diff
@@ -25,6 +25,7 @@
 from pcapi.utils.rest import check_user_has_access_to_offerer
 
 from . import blueprint
+from ...connectors import sirene
```

Comment:
[style] Il devient compliqué de se retrouver dans les imports relatifs quand on dépasse 1 (`from .xxx import`) ou 2 (`from ..x import`) niveaux. Dans ce cas, mieux vaut donner le chemin complet, plus clair (surtout qu'ici il n'est pas beaucoup plus long) : 

```suggestion
from pcapi.connectors import sirene
```